### PR TITLE
Remove meta files, bl scenarios, and added lib

### PR DESCRIPTION
**Pull request by ghost**
## Description

Gets rid of old base game meta files, blacklisted scenarios, and adds lib
## Checklist

- [ x ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [ x ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

**Review by mafewtm — APPROVED** [review]
I'd keep the code and empty config table for the BlacklistedScenarios['TYPES'} so that it can be utilized but there's no need for us to define all those scenarios to blacklist.

**Comment by ghost** [comment]
> I'd keep the code and empty config table for the BlacklistedScenarios['TYPES'} so that it can be utilized but there's no need for us to define all those scenarios to blacklist.

Ok I'll add the empty config back

**Comment by bicatcho11** [comment]
You can only comment on it
In order to make it easier for users to activate or reactivate them instead of using Google to search for each one and what its function is
OR using true false to active

**Comment by bicatcho11** [comment]
I think deleting these files requires more thought
	'popgroups.ymt',
	'relationships.dat'

File: client/ignore.lua [hunk 1]
```diff
@@ -4,7 +4,7 @@ CreateThread(function()
 			SetScenarioTypeEnabled(sctyp, false)
 		end
 		for _, scmdl in next, Config.BlacklistedScenarios['SUPPRESSED'] do
-			SetVehicleModelIsSuppressed(GetHashKey(model), true)
+			SetVehicleModelIsSuppressed(GetHashKey(scmdl), true)
```
Comment (solareon) [inline client/ignore.lua]:
```suggestion
			SetVehicleModelIsSuppressed(joaat(scmdl), true)
```
joaat is faster I believe?

**Review by solareon — CHANGES_REQUESTED** [review]

**Comment by mafewtm** [comment]
> I think deleting these files requires more thought 'popgroups.ymt', 'relationships.dat'

Why would this require more thought?

**Review by solareon — APPROVED** [review]
lgtm

**Pull request merged**